src/components/ValidatedInput/ValidSelect.scss

```diff
@@ -12,3 +12,11 @@
     padding-bottom: 4px;
   }
 }
+
+.styled-atlaskit-select {
```

Comment:
how about moving this to Select/index.scss instead? so that other select will have this style as well

Reply:
Is it safe to do that? If so, then I'll move it

Reply:
It's safe. I've updated the css